app/presenters/email_archive_presenter.rb

```diff
@@ -14,6 +14,7 @@ def for_s3(record, archived_at)
       created_at_utc: record.fetch("created_at").utc.strftime(S3_DATETIME_FORMAT),
       finished_sending_at_utc: record.fetch("finished_sending_at").utc.strftime(S3_DATETIME_FORMAT),
       id: record.fetch("id"),
+      marked_as_spam: true,
```

Comment:
I'm not very familiar with our archiving - does this mean every email sent to S3 has `marked_as_spam` set to `true` or is this hash merged and overridden somewhere else?

Reply:
sorry just noticed that you've fixed this in the PR this depends on. Probably just need a rebase.

Reply:
No worries, thanks for spotting it! Done now!